api/models/Reading.js

```diff
@@ -6,7 +6,23 @@ const readingSchema = new mongoose.Schema({
   value: {
     required: true,
     type: String
+  },
+  tankId: {
+    required: true,
+    type: mongoose.Types.ObjectId
+  },
+  tankCapacity: {
+    required: true,
+    type: Number
+  },
+  tankLocation: {
+    required: true,
+    type: String
```

Comment:
Use [GeoJSON](https://mongoosejs.com/docs/geojson.html)

Reply:
@ArpitKotecha. I have made the necessary changes. Please review.